test/integration/create-next-app/prompts.test.ts

```diff
@@ -1,19 +1,22 @@
-import { join } from 'path'
 import { check } from 'next-test-utils'
+import { join } from 'path'
 import { createNextApp, projectFilesShouldExist, useTempDir } from './utils'
 
-let testVersion
-beforeAll(async () => {
-  // TODO: investigate moving this post publish or create deployed GH#57025
```

Comment:
Might be good to ask JJ if this actually resolves his concern.

Reply:
Yeah exactly